fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java

```diff
@@ -1761,6 +1769,38 @@ Partition createPartition(Database db, OlapTable table, long partitionId, String
         return partition;
     }
 
+    /**
+     * Check and add partitions
+     * NOTE it will modify the PartitionInfo, so it should be proected under WriteLock
+     */
+    private static void checkAndAddPartitions(OlapTable olapTable,
+                                              List<PartitionDesc> partitionDescs,
+                                              AddPartitionClause addPartitionClause,
+                                              PartitionInfo partitionInfo) throws DdlException {
+        Set<String> existPartitionNameSet =
+                CatalogUtils.checkPartitionNameExistForAddPartitions(olapTable, partitionDescs);
```

Comment:
listPartitionInfo.setBatchLiteralExprValues(olapTable.getIdToColumn(), listPartitionInfo.getIdToValues());

The changes in checkPartitionNameExistForAddPartitions do not seem to have any practical significance.